WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -172,6 +196,40 @@ class MySiteViewController: UIViewController, NoResultsViewHost {
         }
     }
 
+    // MARK: - IBAction
+
+    @IBAction func segmentedControlValueChanged(_ sender: Any) {
+        guard let blog = blog else {
+            return
+        }
+
+        if segmentedControl.selectedSegmentIndex == 0 {
```

Comment:
If someone inverts the values of `segmentTitles` this will invert the functionality.

I think we can solve it by using an enum or a different comparison that doesn't rely on the index (that can be easily changed and missed here).

Reply:
Fixed! cb4ba28